schemas/2018-04-16/Microsoft.Insights.json

```diff
@@ -2,21 +2,11 @@
   "id": "https://schema.management.azure.com/schemas/2018-04-16/Microsoft.Insights.json#",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Microsoft.Insights",
-  "description": "Microsoft Insights Resource Types",
+  "description": "microsoft insights Resource Types",
```

Comment:
Why change to lowercase?

Reply:
@anthony-c-martin I can manually change it back to Microsoft Insights - however it is the output from generating with autorest-beta (azureresourceschema version 3.0.67) - wdyt?

Reply:
@adibiton This is originating from the provider namespace in the swagger URI paths ('microsoft.insights' is lower-cased) - e.g. https://github.com/Azure/azure-rest-api-specs/blob/30c4f57844896fe41d0f179f5649760dad9101d4/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json#L272

I'd recommend you fix this for this file + other similar files in the swagger repo going forwards. For now, manually fixing the casing in this file sounds fine to me.